src/ontology/mondo.Makefile

```diff
@@ -226,6 +228,44 @@ reports/%-rare-diseases.tsv: $(ONT)-base.owl reports/%-rare-diseases.txt
 rare-disease-reports: reports/old-rare-diseases.tsv reports/new-rare-diseases.tsv
 	python ../scripts/filter_rare_disease_list.py reports/old-rare-diseases.tsv reports/new-rare-diseases.tsv reports/added-rare-disases.tsv reports/removed-rare-diseases.tsv
 
+
+#########################################
+######### Create Mondo Stats ############
+#########################################
+# Create overview Mondo stats as needed for presentations. This includes: count of disease classes, terms w/definitions, xrefs,
+# exact, related, narrow, and broad synonyms, and count of rare disease, cancer, infectious, and hereditary disease classes.
+
+# Get the current date including the timezone
+current_date := $(shell date)
+
+.PHONY: create-mondo-stats-summary-file create-mondo-stats
+
+# Create the directory if it does not already exist
+$(MONDO_STATS_REPORTS):
+	mkdir -p $(MONDO_STATS_REPORTS)
+
+# Create the Mondo Stats Summary file
+create-mondo-stats-summary-file: reasoned.owl $(MONDO_STATS_REPORTS)
```

Comment:
Instead of this, I tend to do this now:

```suggestion
create-mondo-stats-summary-file: reasoned.owl
	mkdir -p $(MONDO_STATS_REPORTS)
```

 But I forgot why to be honest.

Reply:
I think I've seen you do it both ways

Reply:
After doing some reading, looks like the `|` option is good here. It ensures the directory exists without triggering unnecessary rebuilds when the directory or its contents are modified.